student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
```

Comment:
### Optional
- [ ] It would be better if we use `require_relative` instead of `require` since we are working with the files in our current directory